lib/pagepile_api.rb

```diff
@@ -12,8 +12,7 @@ def page_titles_for_pileid
 
     update_language_and_project
 
-    titles = @pile_data['pages']
-    titles
+    @pile_data['pages']
```

Comment:
Keep this as-is, since it communicates the meaning of the data as a set of page titles.